### ISSUE TITLE
Incorrect or misleading color reported by cell style

### ISSUE THREAD

**Issue by parzivail** [EPPlusSoftware/EPPlus#1337]
I've found a case where the reported color from the style seems to be incorrect. The color shown in Excel is black but the color reported by the `Style.Font.Color` property on the cell is red.

Here's a minimum reproducible example.

	public static void Main(string[] args)
	{
		ExcelPackage.LicenseContext = LicenseContext.NonCommercial;

		using (var package = new ExcelPackage(new FileInfo(@"C:\path\to\test.xlsx")))
		{
			var a2 = package.Workbook.Worksheets[0].Cells["A2"];
			Console.WriteLine(a2.Style.Font.Color.Rgb); // Prints FFFF0000
		}
	}

Here is the color as shown in Excel.
![image](https://github.com/EPPlusSoftware/EPPlus/assets/8730127/f9d7a881-0658-4a16-9624-64e431ecc378)

Here's the workbook that can reproduce the issue, created with Office 2021.
[test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/14546754/test.xlsx)

I can't figure out if this is an issue with my workbook or with EEPlus, but it's inconsistent nonetheless. If it's possible to get a workbook into this state in Excel, I would like to prevent that, or at least understand how to work around it.

**Comment by JanKallman** [comment]
I think this issue might be the same as #1320 .
This issue is fixed in EPPlus 7.0.10.

**Comment by parzivail** [comment]
Thanks! I'll give it a shot this afternoon and let you know.

**Comment by parzivail** [comment]
Unfortunately this does not seem to be resolved with 7.0.10. I'm targeting .NET 8 if it makes a difference.

**Comment by parzivail** [comment]
For completeness, I just tested as far back as 4.5.1 and no version seems to report the color correctly, which leads me wonder if/how the spreadsheet is malformed.

**Comment by JanKallman** [comment]
Looking at A2 in you workbook it has no StyleId Set...
```Xml
<row r="2" spans="1:4" s="4" customFormat="1" x14ac:dyDescent="0.25">
<c r="A2" t="s">
<v>2</v>
</c>
<c r="B2"/>
<c r="C2"/>
<c r="D2"/>
</row>
```
This means A2 will reference to the xsfId 0 - fontId 0, in the styles.xml:

```Xml
<font>
<sz val="11"/>
<color theme="1"/>
<name val="Calibri"/>
<family val="2"/>
<scheme val="minor"/>
</font>
```¨
theme="1" - is Text 1 - Black
Did you create this workbook with EPPlus? In that case can you provide the code necessary to create it?

**Comment by parzivail** [comment]
No, this was made in Excel 2021

**Comment by JanKallman** [comment]
Yes, you are right. This is a bug. 
When having no StyleId on the cell, EPPlus takes the style from the row or the column. In this case the row. As the cell refers to StyleId 0, the wrong color is returned (FFFF0000 instead of theme Text1). I will provide a fix.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.1.1
